src/assets/ElmAsset.js

```diff
@@ -0,0 +1,43 @@
+const JSAsset = require('./JSAsset');
+const localRequire = require('../utils/localRequire');
+const process = require('process');
+
+// Currently no pretty way to install Elm tools on behalf of the user :/
+async function compile(name, options) {
```

Comment:
Can we just inline this function into `parse` below?

Reply:
Will do